deny.toml

```diff
@@ -126,3 +90,22 @@ unknown-git = "warn"
 #allow-registry = []
 # List of URLs for allowed Git repositories
 allow-git = []
+
+[graph]
+# If 1 or more target triples (and optionally, target_features) are specified, only
+# the specified targets will be checked when running `cargo deny check`. This means,
+# if a particular package is only ever used as a target specific dependency, such
+# as, for example, the `nix` crate only being used via the `target_family = "unix"`
+# configuration, that only having windows targets in this list would mean the nix
+# crate, as well as any of its exclusive dependencies not shared by any other
+# crates, would be ignored, as the target list here is effectively saying which
+# targets you are building for.
+targets = [
+    # The triple can be any string, but only the target triples built in to
+    # rustc (as of 1.40) can be checked against actual config expressions
+    #{ triple = "x86_64-unknown-linux-musl" },
+    # You can also specify which target_features you promise are enabled for a particular
+    # target. target_features are currently not validated against the actual valid
+    # features supported by the target architecture.
+    #{ triple = "wasm32-unknown-unknown", features = ["atomics"] },
+]
```

Comment:
I guess this is just kept to document the feature?

Reply:
It's probably copy-pasted from the template that cargo-deny generates